string file_path
int8 image_count
